pkg/controller/zookeepercluster/zookeepercluster_controller.go

```diff
@@ -366,3 +371,83 @@ func (r *ReconcileZookeeperCluster) reconcileClusterStatus(instance *zookeeperv1
 		"StatefulSet.Name", instance.Name)
 	return r.client.Status().Update(context.TODO(), instance)
 }
+
+// YAMLExporterReconciler returns a fake Reconciler which is being used for generating YAML files
+func YAMLExporterReconciler(zookeepercluster *zookeeperv1beta1.ZookeeperCluster) *ReconcileZookeeperCluster {
+	var scheme = scheme.Scheme
+	scheme.AddKnownTypes(zookeeperv1beta1.SchemeGroupVersion, zookeepercluster)
+	return &ReconcileZookeeperCluster{
+		client: fake.NewFakeClient(zookeepercluster),
+		scheme: scheme,
+	}
+}
+
+// GenerateYAML generated secondary resource of ZookeeperCluster resources YAML files
+func (r *ReconcileZookeeperCluster) GenerateYAML(inst *zookeeperv1beta1.ZookeeperCluster) error {
+	// Fetch the ECSCluster inst
```

Comment:
I think the comment is inaccurate. Probably unnecessary since we're logging what's happening two lines later.

Reply:
All review comments are incorporated.